lighthouse-core/audits/manifest-display.js

```diff
@@ -53,9 +53,14 @@ class ManifestDisplay extends Audit {
     return ManifestDisplay.generateAuditResult({
       rawValue: hasRecommendedValue,
       displayValue,
-      debugString: 'Manifest display property should be set.'
+      debugString: this.createDebugString(hasRecommendedValue)
```

Comment:
Ah, I was a little confused between what @brendankenny mentioned below. 

> I think it would be better to just inline a test for debugString to exist or not in the above tests instead of adding new tests just for debugString

Now, the property is absent from the object if valid, i.e. what you pasted above.